extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -932,7 +932,12 @@ export class PublishDatabaseDialog {
 	* Gets the default deployment options from the dacfx service
 	*/
 	public async getDefaultDeploymentOptions(): Promise<DeploymentOptions> {
-		return await utils.getDefaultPublishDeploymentOptions(this.project) as DeploymentOptions;
+		let defaultDeploymentOptions = await utils.getDefaultPublishDeploymentOptions(this.project) as DeploymentOptions;
+		if (defaultDeploymentOptions !== undefined && defaultDeploymentOptions.excludeObjectTypes !== undefined) {
+			// For publish dialog no default exclude options should exists
+			defaultDeploymentOptions.excludeObjectTypes.value = [];
```

Comment:
this is different from the previous behavior. What object types besides DatabaseScopedCredentials are now being included?

Reply:
As discussed and as per SSDT no object types are excluding for the Publish. Thanks.